packages/venia-ui/lib/components/RichContent/richContent.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import detectPageBuilder from './PageBuilder/detectPageBuilder';
+import PageBuilder from './PageBuilder';
+import { mergeClasses } from '../../classify';
+import defaultClasses from './richContent.css';
+import { shape, string } from 'prop-types';
+
+const toHTML = str => ({ __html: str });
+
+/**
+ * RichContent component.
+ *
+ * This component serves as the pool to determine which type of content is being rendered
+ * and pass the data off to the correct system.
+ *
+ * @typedef RichContent
+ * @kind functional component
+ *
+ * @param {Object} props React component props
+ *
+ * @returns {React.Element} A React component that renders Heading with optional styling properties.
+ */
+const RichContent = props => {
+    const { html } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    if (detectPageBuilder(html)) {
```

Comment:
It occurs to me that not everyone will have PageBuilder but everyone using PWA Studio will get _all_ the PageBuilder code bundled, right? Or do you do do some lazy load? I think you must...

Reply:
You're right, if someone isn't using Page Builder but does invoke `<RichContent />` they'll pull down some of Page Builder (well in fact it'll be in the bundle).

This was _somewhat intentional_ as if Page Builder is used on the store we want to ensure important content is loaded _as quickly as possible_. We want to avoid huge jumps in content loads. Including `<PageBuilder />` with a `React.lazy` could solve this, but I believe that'd make the rendering of the actual pages content delayed (even if just by a few ms).

Scenario: a homepage with content on it. The content is the most important part of said page, so it should have priority for the render. 

@zetlen we discussed around this topic a little bit, would love to hear your views here. Should we just `React.lazy` the whole `<PageBuilder />` component? or is the current situation okay?

Reply:
Spoke about this with @zetlen and using `stats:venia` we identified that PageBuilder will only be included in the CMS Page chunk. We agreed that we shouldn't implement something now to resolve this and instead wait until we have a better extensibility model which will allow us to disable / enable Page Builder.